### ISSUE TITLE
Emoji Support seems broken in the latest WeasyPrint

### ISSUE THREAD

**Issue by oz123** [Kozea/WeasyPrint#1506]
I am trying to create a  PDF with the following content:
```
$ minikube node add -p devopscon
😄  Adding node m02 to cluster devopscon
❗  Cluster was created without any CNI, adding a node to it might cause broken networking.
👍  Starting worker node devopscon-m02 in cluster devopscon
🔥  Creating kvm2 VM (CPUs=2, Memory=2848MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.22.3 on Docker 20.10.8 ...
🔎  Verifying Kubernetes components...
🏄  Successfully added m02 to devopscon!
```

I document is built with: 
```
easyprint -e utf-8 _build/singlehtml/index.html "_build/output.pdf"
```

The header contains:

```
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />
...
```
The emojis are shown in this version:
```
 $ weasyprint --version
WeasyPrint version 52.5
```

The emojis are not showing with the latest version,
```
$ weasyprint --version
WeasyPrint version 53.4
```

**Comment by oz123** [comment]
Woot! This works! On the master branch, and it look much better than the ugly png based rendered emojis !!! :clap:

**Comment by liZe** [comment]
> Woot! This works! On the master branch, and it look much better than the ugly png based rendered emojis !!! :clap:

Yes, it’s been fixed by #1434, and we now handle SVG emojis that look better than raster ones 😁.
